includes/Admin/Settings_Screens/Messenger.php

```diff
@@ -156,6 +157,34 @@ public function get_settings() {
 	}
 
 
+	/**
+	 * Gets the upcoming deprecation warning notice.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string
+	 */
+	public function get_deprecation_warning() {
+		return '<div class="notice notice-warning"><p>' . self::get_deprecation_message() . '</p></div>';
+	}
+
+	/**
+	 * Gets the upcoming deprecation warning message with documentation link.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string
+	 */
+	public static function get_deprecation_message() {
+		return sprintf(
+			/* translators: Placeholders: %1$s - <a> tag, %2$s - </a> tag */
+			__( '️<strong>Facebook for WooCommerce</strong>: Since %1$sMeta plans to discontinue the Chat Plugin%2$s, the Messenger feature will be removed by May, 2024.', 'facebook-for-woocommerce' ),
+			'<a href="https://href.li/?https://www.facebook.com/business/help/1524587524402327" target="_blank">',
```

Comment:
Do we really need href.li?
```suggestion
			'<a href="https://www.facebook.com/business/help/1524587524402327" target="_blank">',
```

Reply:
Whoops! Copied it off my notes, must have had that on the link. Thanks 👍🏻